.github/workflows/test.yml

```diff
@@ -0,0 +1,70 @@
+name: Test
+
+on:
+  pull_request:
```

Comment:
Couldn't we define `paths` here so that this workflow only runs when CI files are modified?


E.g.:

```yaml
on:
  pull_request:
    .github/actions/**
    .github/workflows/*.yml
    ci/**
```

---

This would work in tandem with the `prepare` job below.